modules/RTC/RTC.js

```diff
@@ -15,6 +14,8 @@ import VideoType from '../../service/RTC/VideoType';
 
 const logger = getLogger(__filename);
 
+let rtcTrackIdCounter = 0;
```

Comment:
it is used from static method of RTC, is it possible to make this var static ? I don't know how to make it better